File: docs-site/python/reconnect_handler.py
from relayx_py import Realtime

async def on_reconnect(status):
    if status == "RECONNECTING":
        print("Looks like we're not connected, attempting to reconnect to Relay...")
    elif status == "RECONNECTED":
        print("Reconnected to Relay! :D")
    elif status == "RECONN_FAIL":
        print("Failed to reconnect to Relay :(")

await client.on(Realtime.RECONNECT, on_reconnect)

File: docs-site/android/reconnect_handler.kt
import com.realtime.relay.realtimeSDK.Realtime

client.on(Realtime.RECONNECT) { status ->
    if(status == Realtime.RECONNECTING){
        Log.d("Relay", "Looks like we're not connected, attempting to reconnect to Relay...")
    }else if(status == Realtime.RECONNECTED){
        Log.d("Relay", "Reconnected to Relay! :D")
    }else if(status == Realtime.RECONN_FAIL){
        Log.d("Relay", "Failed to reconnect to Relay :(")
    }
}